### PR TITLE
Update notification in submission_list_view

### PR DESCRIPTION
**Pull request by classPythonAddike**
When submissions have not opened, and someone clicks on the `View Submissions` button on the timathon page, it shows an alert on the top right hand corner saying that voting hasn't started yet. I changed it to `Submissions haven't closed yet` :mhmf:

### CODE REVIEW

File: TWT/apps/timathon/views/submission_list_view.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def get(self, request: WSGIRequest):
             not context["is_staff"] and challenge.team_creation_status
         ):
             messages.add_message(
-                request, messages.INFO, "The voting period is not open please wait"
+                request, messages.INFO, "Submissions have not closed yet, please wait for the same!"
```
Comment (takos22) [inline TWT/apps/timathon/views/submission_list_view.py:43]: `please wait` is enough imo, no need for a long sentence

**Comment by SylteA** [comment]
Hey @classPythonAddike 

Will you complete this PR?

**Comment by classPythonAddike** [comment]
Sure :mhm:

**Comment by Shubhaankar-Sharma** [comment]
Hey @classPythonAddike Please keep it clean, we want to branch from Tech-With-Tim:dev since there are already commits on that branc, after that we can merge it to master at once

**Comment by classPythonAddike** [comment]
ok

**Pull request closed without merging**